httpreplay/httpreplay.go

```diff
@@ -57,6 +57,16 @@ func NewRecorder(filename string, initial []byte) (*Recorder, error) {
 	return &Recorder{proxy: p}, nil
 }
 
+// ScrubBody will replace all parts of the request body that match any of the
```

Comment:
`ClearBody`, for consistent terminology?

Reply:
Kept as Scrub due to the different semantics.